data/tests/tests_common.txt

```diff
@@ -585,6 +585,15 @@ test "Depart"
 			"test: Depart: zoom_restore_counter" > 0
 
 
+test "Land"
+	status partial
+	description "Helper ffunction to land on the first selected planet. If that planet does not allow the ship to land, the test will fail."
+	sequence
+		input
+			key l
+		watchdog 12000
```

Comment:
This function sends the key-input for landing, but does not actually have the test-code wait until the ship has landed. (The watchdog is a "safety timeout" to avoid hanging forever in a hanging test, but it is not a wait-loop.) Any inputs given after this land-function will not happen on-planet, but will likely be given while the ship still is busy landing.

I usually use the `"flagship planet: <planetname>"` condition together with a branch to [wait until the flagship has landed](https://github.com/warp-core/endless-sky/blob/4d4f4a1a4099cf82a5165d1a87d012e5256b768b/data/tests/tests_framework.txt#L121).